rules-xml/src/main/java/org/jboss/windup/rules/apps/xml/confighandler/XmlFileHandler.java

```diff
@@ -51,7 +53,7 @@ public XmlFile processElement(ParserContext handlerManager, Element element)
         {
             throw new WindupException("Error, 'xmlfile' element must have a non-empty 'matches' or public-id attribute");
         }
-        String inFile = $(element).attr("in-file");
```

Comment:
in is used in the JavaClass also and all of our xml rules were using "in" and not "in-file", so I switched it here